### ISSUE TITLE
Moving scene

### ISSUE THREAD

**Issue by chiyaohuang** [Asad-Shahid/Intelligent-Task-Learning#2]
Hi Asad,

I am really interested in your work, but I faced some problems when running it.
1. When I run the default mode "$ python -m rl.main", the robot does not clip the cube successfully. Should I increase "--num_eval"?
2. The moving object grasping showed in your video seems not to define in "--mode". Will you update it in the future?

Sincerely,
Yao

**Comment by Asad-Shahid** [comment]
Hi, 
If you just run:
```
python -m rl.main
```
the evaluation mode is run by default which means the agent executes the random policy that cannot grasp the cube. 
In order to learn grasping, you need to train the agent. I have also updated the repo with the pre-trained policy that grasps the cube. 

2. The moving object experiment was done for a paper. It is not included in the repo currently but it is a slight adjustment and you only need to specify the velocity of the object in simulation.

**Comment by chiyaohuang** [comment]
I just ran up your pretrained model. It looks awesome! I am looking forward to your paper!

**Comment by Asad-Shahid** [comment]
Here is the [link](https://link.springer.com/article/10.1007/s10514-022-10034-z) to the recently published paper
